app/styles/theme.css

```diff
@@ -33,6 +33,20 @@ body {
 /* Material Design ("MD") colors from
    https://www.google.com/design/spec/style/color.html#color-color-palette */
 
+noscript {
+  justify-content: center;
+}
+
+noscript p {
+  max-width: 240px;
+  margin: 10px auto 10px auto;
+  padding: 20px;
+  border-radius: 3px;
+  background-color: #eeeeee;    /* MD Grey 200*/
+  font-size: 5vh;
+  text-align: center;
+}
```

Comment:
The purpose of `theme.css` is to apply styling like colors and spacing all in one place, and this centralization is supposed to help keep that styling consistent.  Part of this is due to the decision to use plain CSS, which is half tied up with the project's history, and half as an experiment.

Anyway, properties declared here that you see declared elsewhere in this file should stay here, and probably the rest should be moved to a file called `noscript.css`.

You may also notice that for essentially all other styles where it was possible, I opted to use classes instead of element selectors.  I believe this is a more reliable way of writing CSS for a web app, and would prefer if these styles could be declared under class selectors instead.